extensions/blocks/videopress/save.js

```diff
@@ -21,6 +21,9 @@ const VideoPressSave = CoreVideoSave => props => {
 			poster,
```

Comment:
Since we're changing the structure of the saved data, we'll need to handle deprecation for previous version of the blocks:
https://developer.wordpress.org/block-editor/developers/block-api/block-deprecation/

Reply:
It seems to be validating ok for me if I create an older version of the block, compile this branch and then load the block again. Is it because these color vars have default values? The data in the block will only change if the embed url updates, I think.

Reply:
Yeah, you're right, it seems to work just fine! 👍